app/helpers/date.py

```diff
@@ -0,0 +1,75 @@
+import datetime
```

Comment:
think we should scrap the helpers folder and only use utils folder. Both are synonymous I think

Reply:
OK when we merge everything we should apply this, because now we have 4 branaches depending on the helper folder.